flake.nix

```
@@ -13,23 +13,44 @@
       url = "github:sellout/bash-strict-mode";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+
+    home-manager = {
+      url = "github:nix-community/home-manager/release-24.05";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
   };
 
-  outputs = inputs:
+  outputs = inputs: let
+    systems = inputs.nixpkgs.lib.systems.flakeExposed;
```

Doesn't seem like we can actually support all these systems. How about setting it to something more conservative for now?
```suggestion
    systems = ["x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin"];
```

---

Ah, I missed this failure somehow.

So, these failures are actually because of garnix-io/issues#94 – it’s _CI_ that doesn’t support these systems, they may very well build agenix.el just fine. Granted, without a way to check that they actually work, including these doesn’t really help anyone.

I think we should support the four that garnix will actually check, plus `x86_64-darwin` (I can test it locally on my M1 Mac, via Rosetta 2, but [garnix has apparently had issues with that system](https://github.com/garnix-io/issues/issues/16#issuecomment-1962401335).)

```suggestion
    systems = [
      "aarch64-darwin"
      "aarch64-linux"
      "i686-linux"
      "x86_64-darwin"
      "x86_64-linux"
    ];
```

This is the default set I use on my own projects, but including `x86_64-darwin` _does_ require adding a garnix.yaml file to explicitly exclude the `homeConfiguration` from CI.

---

Alright, it might be a good workaround to use your list of systems and for now exclude `x86_64-darwin` build